docs/troubleshooting.md

```diff
@@ -0,0 +1,35 @@
+# Troubleshooting
+
+## NGINX and proxying
+
+While outside the direct responsibility of this project we get a lot of issues reported that relate to proxying with 
+NGINX into the FPM container.
+
+Note that you will need to set the name of your NGINX container to be in the list of TRUSTED_HOSTS when you start the 
```

Comment:
The whitespaces after `"the "` violates markdownlint rule [MD009 - Trailing spaces](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md009).